presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleDistributedQueries.java

```diff
@@ -0,0 +1,420 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.Session;
+import io.prestosql.testing.AbstractTestDistributedQueries;
+import io.prestosql.testing.MaterializedResult;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.testing.sql.TestTable;
+import io.prestosql.tpch.TpchTable;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static io.prestosql.spi.type.VarcharType.VARCHAR;
+import static io.prestosql.testing.MaterializedResult.resultBuilder;
+import static java.lang.String.format;
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.IntStream.range;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class TestOracleDistributedQueries
+        extends AbstractTestDistributedQueries
+{
+    private OracleServer oracleServer;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        this.oracleServer = new OracleServer();
+        return OracleQueryRunner.createOracleQueryRunner(
+                oracleServer,
+                ImmutableMap.<String, String>builder().build(),
+                TpchTable.getTables());
+    }
+
+    @AfterClass(alwaysRun = true)
+    public final void destroy()
+    {
+        if (oracleServer != null) {
+            oracleServer.close();
+        }
+    }
+
+    @Override
+    protected boolean supportsViews()
+    {
+        return false;
+    }
+
+    @Override
+    protected boolean supportsArrays()
+    {
+        return false;
+    }
+
+    @Override
+    public void testCommentTable()
+    {
+        // table comment not supported
+    }
+
+    @Override
+    public void testDelete()
+    {
+        // delete is not supported
+    }
+
+    @Override
+    public void testCreateSchema()
+    {
+        // schema creation is not supported
```

Comment:
While the current code throws the following exception, the syntax itself is supported in BaseJdbcClient.
If we disallow it, it would be better to override `createSchema` in OracleClient and throw the exact exception. 
```
java.lang.RuntimeException: ORA-02420: missing schema authorization clause
```

Reply:
done